integration/airflow/openlineage/airflow/facets.py

```diff
@@ -25,3 +25,9 @@ def from_task(cls, task):
 @attr.s
 class AirflowRunArgsRunFacet(BaseFacet):
     externalTrigger: bool = attr.ib(default=False)
+
+
+@attr.s
+class ExternalQueryJobFacet(BaseFacet):
```

Comment:
Since `SqlJobFacet` is defined as having a `query` field, I would be consistent with the naming. That is,

```
ExternalSqlJobFacet:
   query: string
   queryId: string
   source: string
```

Also, if the query is external I would set `ExternalSqlJobFacet.query` = `SQL` value and not set the `SqlJobFacet`. It would make sense that it would be on or the other i.e. either external or not

Reply:
Also, it fits more as a run facet - the query id changes with every run, while SQL usually is the same between multiple runs of the same job.